css/src/components/hero.scss

```diff
@@ -30,12 +30,15 @@ $hero-image-width-widescreen: calc((100% - 2 * $layout-widescreen-gap) * 0.55) !
 			width: $hero-content-fluid-width;
 			min-height: $hero-min-height-default;
 			padding-block: $hero-padding-default;
-			padding-inline-end: $hero-padding-sm;
+			// not using logical properties here to avoid spacing issues for RTL
+			padding-right: $hero-padding-sm; /* stylelint-disable-line csstools/use-logical */
```

Comment:
This works for sure, but I'm wondering if you considered instead of removing logical prop from here, adding these to line 150?

```
.hero-content {
	direction: rtl;
	padding-inline-end: 0;
	padding-inline-start: $hero-padding-sm;
}

```

![image](https://github.com/microsoft/atlas-design/assets/57374379/029e141c-6ee6-4435-b9c7-4c0e879b85f4)

Reply:
Yes, I also considered the suggested change above. Both would work. I updated to the above, but wrapped it in the desktop media query to match.